test/bdd/fixtures/docker-compose.yml

```diff
@@ -582,7 +588,7 @@ services:
       - KMS_LOG_LEVEL=debug
     ports:
       - 7878:7878
-    entrypoint: ""
+    entrypoint: [""]
```

Comment:
What is this change for?

Reply:
Sorry, updated the PR comment:

Also fixed the orb-kms container to work with Docker 3.5.2.

I updated Docker and orb-kms wouldn't start.